lib/mock/mock-utils.js

```diff
@@ -107,9 +107,9 @@ function getMockDispatch (mockDispatches, key) {
 }
 
 function addMockDispatch (mockDispatches, key, data) {
-  const baseData = { times: null, persist: false, consumed: false }
+  const baseData = { timesInvoked: 0, times: 1, persist: false, consumed: false }
   const replyData = typeof data === 'function' ? { callback: data } : { ...data }
-  const newMockDispatch = { ...baseData, ...key, data: { error: null, ...replyData } }
+  const newMockDispatch = { ...baseData, ...key, pending: true, data: { error: null, ...replyData } }
```

Comment:
All interceptors are `pending` as they are first registered. I think `consumed` belongs here as well (and not in `baseData`), but… meh. I suppose it might be possible to register an interceptor that should not be `pending` by registering it with `.times(0)`, but that sounds too illogical to account for.